### PR TITLE
Created Recipe card Component 

### PR DESCRIPTION
**Pull request by strawberrie68**
Created RecipeCard Component 
- Component renders main info about the recipe

Also attached test for RecipeCard Component 

**Updated Recipe Card**
Screenshot 2023-11-07 at 2 34 11 pm

### CODE REVIEW

File: src/components/TypesOfRecipeCards/RecipeCard.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                  className="h-8 w-8 rounded-full"
+                />
+                <p className=" ml-2 text-xxxs">
+                  by {`${recipe.userId.firstName} ${recipe.userId.lastName}`}
+                </p>
+              </div>
+              <div className="ml-2 mt-1 flex items-baseline">
+                <div className="text-xxxs ">{recipe.rating} ⭐️</div>
+                <div className="ml-2 text-xxxs font-thin">
+                  {recipe.review.length} Reviews{" "}
```
Comment (laurawong) [inline src/components/TypesOfRecipeCards/RecipeCard.jsx]:
```suggestion
                  {recipe.review.length} Reviews{" "}
```
What is `{" "}` for?

File: src/components/TypesOfRecipeCards/RecipeCard.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                </div>
+              )}
+            </div>
+
+            <img
+              src={recipe.picturePath}
+              className="h-full w-full rounded-3xl object-cover"
+            />
+            <div className="absolute bottom-2 ml-4 flex flex-col text-white">
+              <div className="flex items-baseline	">
```
Comment (laurawong) [inline src/components/TypesOfRecipeCards/RecipeCard.jsx]:
```suggestion
              <div className="flex items-baseline">
```
Remove whitespaces from this file

File: src/components/TypesOfRecipeCards/RecipeCard.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                <img
```
Comment (laurawong) [inline src/components/TypesOfRecipeCards/RecipeCard.jsx:30]: UX issue: profile picture blends into recipe picture

File: cypress/component/RecipeCard.cy.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      id: "1234",
+      firstName: "Anna",
+      lastName: "Real",
+      picturePath: User1,
+    },
+  };
+
+  it("all the user info should be rendered", () => {
+    cy.mount(<RecipeCard recipe={recipe} />);
+    cy.contains(recipe.userId.firstName + " " + recipe.userId.lastName);
```
Comment (laurawong) [inline cypress/component/RecipeCard.cy.jsx]:
```suggestion
    cy.contains(`${recipe.userId.firstName recipe.userId.lastName`);
```
I think it's a bit easier to read like this

File: cypress/component/RecipeCard.cy.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import RecipeCard from "../../src/components/TypesOfRecipeCards/RecipeCard";
+import Pizza from "../../src/assets/pizza.jpg";
+import User1 from "../../src/assets/user/user1.svg";
+
+describe("Render RecipeCard componenet ", () => {
+  const recipe = {
+    title: "Itallian Pizza",
```
Comment (laurawong) [inline cypress/component/RecipeCard.cy.jsx]:
```suggestion
    title: "Italian Pizza",
```

File: src/components/TypesOfRecipeCards/RecipeCard.jsx (hunk 3 above)
Comment (strawberrie68) [reply src/components/TypesOfRecipeCards/RecipeCard.jsx:30]: Do you think a white border would help?
Added a white border to the profile picture to the updated picture above.

File: cypress/component/RecipeCard.cy.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+      firstName: "Anna",
+      lastName: "Real",
+      picturePath: User1,
+    },
+  };
+
+  it("all the user info should be rendered", () => {
+    cy.mount(<RecipeCard recipe={recipe} />);
+    cy.contains(`${recipe.userId.firstName} ${recipe.userId.lastName}`);
+    cy.contains(recipe.rating + " ⭐️");
```
Comment (laurawong) [inline cypress/component/RecipeCard.cy.jsx:25]:
```suggestion
    cy.contains(`${recipe.rating ⭐️}`);
```

File: cypress/component/RecipeCard.cy.jsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    cy.contains(recipe.review.length + " Reviews");
```
Comment (laurawong) [inline cypress/component/RecipeCard.cy.jsx:26]:
```suggestion
    cy.contains(`${recipe.review.length Reviews`);
```

File: cypress/component/RecipeCard.cy.jsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      id: "1234",
+      firstName: "Anna",
+      lastName: "Real",
+      picturePath: User1,
+    },
+  };
+
+  it("all the user info should be rendered", () => {
+    cy.mount(<RecipeCard recipe={recipe} />);
+    cy.contains(`${recipe.userId.firstName} ${recipe.userId.lastName}`);
```
Comment (laurawong) [inline cypress/component/RecipeCard.cy.jsx:24]:
```suggestion
    cy.mount(<RecipeCard recipe={recipe} />);
  
    cy.contains(`${recipe.userId.firstName} ${recipe.userId.lastName}`);
```

Add whitespace so reader can see what's being set up and asserting at a quick glance.

**Review by laurawong — APPROVED** [review]

File: cypress/component/RecipeCard.cy.jsx (hunk 6 above)
Comment (strawberrie68) [reply cypress/component/RecipeCard.cy.jsx:25]: I don't know why but cypress don't like this

**Pull request merged**